README.rst

```diff
@@ -8,34 +8,36 @@
 .. image:: https://img.shields.io/badge/licence-GPL_2-green.svg?style=flat
    :target: https://raw.githubusercontent.com/cernopendata/opendata.cern.ch/master/LICENSE
 
-.. image:: https://badges.gitter.im/Join%20Chat.svg
-   :target: https://gitter.im/cernopendata/opendata.cern.ch?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
-
 About
 -----
 
-This is the source code behind CERN Open Data portal. You can access the portal
-at `http://opendata.cern.ch/ <http://opendata.cern.ch>`_.  The source code uses
-`Invenio <https://inveniosoftware.org/>`_ digital repository framework.
+This repository is part of for the `CERN Open Data portal
+<https://opendata.cern.ch>`_, hosting the source versions of the content,
+notably data records and associated documentation pages. The CERN Open Data
+portal's infrastructure code is living in the sister `cernopendata-portal
```

Comment:
```suggestion
portal's infrastructure code is living in the `cernopendata-portal
```

Reply:
Removed here and in two other places in the developing guide.